README.md

```diff
@@ -68,7 +68,9 @@ You can run the following command to test omni with that repository:
 omni clone https://github.com/omnicli/omni-example-go.git
 ```
 
-## Configuration
+## Setting up omni for your repo
+
+Omni uses an `.omni.yaml` file at the top-level in your repo to determine how to configure omni's management of your repo.
```

Comment:
This is actually not complete!
Omni uses a number of potential configuration files, some local (.omni or .omni.yaml or .omni/config or .omni/config.yaml from the root of the repo) and some "global" (per user) located at ~/.omni or ~/.omni.yaml or ~/.config/omni or ~/.config/omni.yaml, depending on preferences.

The configuration options presented here can be set `per repo` or for the user.
Some of those configuration options do not make sense to use as global options, while some others don't make sense to use as local options!

I do want to convert the whole documentation into a docusaurus stuff or equivalent though to make it easier to navigate than a simple README...

Reply:
A bigger and more fleshed out docs would make a lot of sense, but for the moment: just hoping to fill in the sort of details at least someone like me wants to know out of the gate to get an idea of how the tool works.